### PR TITLE
feat: Add a small paragraph of text below 'Featured projects' and 'credit classes' on the homepage

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1398

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

Go to https://deploy-preview-1620--regen-registry.netlify.app/

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/molecules/BlockContentBody/BlockContentBody.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export interface Props {
+  body: Scalars['JSON'];
+}
+
+const BlockContentBody = ({ body }: Props): JSX.Element => (
+  <Body
+    size="xl"
+    textAlign="center"
+    sx={{ margin: '0 auto', maxWidth: 800, pb: { xs: 12.5, sm: 12.5 } }}
```
Comment (blushi) [inline web-registry/src/components/molecules/BlockContentBody/BlockContentBody.tsx:14]: still waiting for confirmation whether we want the same pb on mobile/desktop or not

Comment (blushi) [inline web-registry/src/components/molecules/BlockContentBody/BlockContentBody.tsx:14]:
```suggestion
    sx={{ margin: '0 auto', maxWidth: 800, pb: { xs: 7.5, sm: 12.5 } }}
```

**Review by flagrede — APPROVED** [review]
Looks good !

**Review by wgwz — APPROVED** [review]

**Review by mhagel — APPROVED** [review]

**Comment by blushi** [comment]
@erikalogie could you review this please?

**Comment by erikalogie** [comment]
This looks great! Just one small detail that the "projects" and "Credit classes" headers on mobile look better with less space between them and the paragraph: https://monosnap.com/file/5UXNqh7EhQF0lAJ91wRkcSuCRO8aDf

**Comment by blushi** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**